site/docs/pages/xmtp/introduction.mdx

```diff
@@ -1,67 +0,0 @@
----
-title: Introduction to XMTP utilities · OnchainKit
```

Comment:
So this intro actually is kind of useful..... So we might want to move this content somewhere else.

Reply:
Or we need to improve the docs on the XMTP pages.

Reply:
Let's leave the `XMTP introduction` page for now until we have a `XMTP Guide` that will include this important information and ultimately replace this page.